ParisBNF/abb/BNFabb174.xml

```diff
@@ -0,0 +1,342 @@
+<?xml version="1.0" encoding="UTF-8"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng"
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng"
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="BNFabb174" xml:lang="en" type="mss">
+   <teiHeader xml:base="https://betamasaheft.eu/">
+      <fileDesc>
+         <titleStmt>
+            <title>Sheperd of Hermas</title>
+            <editor role="cataloguer" key="DN"/>
+            <editor role="generalEditor" key="AB"/>
+         </titleStmt>
+         <publicationStmt>
+            <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+            <pubPlace>Hamburg</pubPlace>
+            <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+               Forschungsumgebung / Beta maṣāḥǝft</publisher>
+            <availability>
+               <licence target="http://creativecommons.org/licenses/by-sa/4.0/">
+                  <p> This file is licensed under the Creative Commons Attribution-ShareAlike 4.0.
+                  </p>
+               </licence>
+            </availability>
+            <date>2016-06-07T17:45:12.922+02:00</date>
+         </publicationStmt>
+         <sourceDesc>
+               <msDesc xml:id="ms">
+              <msIdentifier>
+                  <repository ref="INS0303BNF"/>
+                  <collection>Manuscrits orientaux</collection>
+                  <collection>Fonds éthiopien</collection>
+                  <collection>d'Abbadie</collection>
+                <idno facs="https://gallica.bnf.fr/ark:/12148/btv1b10087926g/f1.item.r=Abbadie%20174#">BnF Éthiopien d'Abbadie 174</idno>
```

Comment:
note also the facs is wrong
should be 
https://gallica.bnf.fr/view3if/ga/ark:/12148/btv1b10087926g

to get the correct url always click on the IIIF symbol
![image](https://user-images.githubusercontent.com/17987728/161025525-ac0a612e-3e85-4900-bbf4-5a17fa50dace.png)